File: C/Chapter-07/Question/Question3.c
#include <stdio.h>

int main(){
  int sum=0;
  int num;

  while(1){
    printf("Enter number: ");
    scanf("%d", &num);
    sum += num;
    if (num == 0) break;
  }
  printf("sum = %d\n", sum);

  return 0;
}

File: Java/Chapter-05/Question/Question3.java
public class Question3 {
  public static void main(String[] args) {
    int num = 1, sum = 0;
    while(num < 1000){
      if((num % 2==0) && (num % 7==0)){
        System.out.print(num + " ");
        sum += num;
      }
      num++;
    }
    System.out.println("\nsum = " + sum);
  }
}
